src/compiler/types.ts

```diff
@@ -1543,6 +1543,13 @@ module ts {
         numberIndexType: Type;             // Numeric index type
     }
 
+    /* @internal */

```

Comment:
Seeing as `@internal` is a pseudo-modifier, yeah, let's move it a little closer.